example/demo/src/app.rs

```diff
@@ -179,17 +166,23 @@ fn HomePage() -> impl IntoView {
                                         <p>"press 'Escape' to reset the count"</p>
                                     </div>
                                 </div>
+                                <div
+                                    _ref=a_ref
+                                    class:green=move || is_green.get() == true
+                                    class:yellow=move || is_green.get() == false
```

Comment:
you need to have `tabIndex=-1` for `_ref` attribute to be clickable/work... take a look at: https://react-hotkeys-hook.vercel.app/docs/documentation/useHotkeys/scoping-hotkeys

Reply:
Gotcha -- I'll make sure to note this in our Readme